packages/okta-react/test/e2e/harness/src/Home.js

```diff
@@ -60,11 +59,17 @@ export default withAuth(class Home extends Component {
       <button id="logout-button" onClick={this.logout}>Logout</button> :
       <button id="login-button" onClick={this.login}>Login</button>;
 
+    const pkce = this.props.auth._oktaAuth.options.pkce;
+
     return (
       <div>
+        <div id="login-flow">{ pkce ? 'PKCE' : 'implicit'}</div>
+        <hr/>
         <Link to='/'>Home</Link><br/>
         <Link to='/protected'>Protected</Link><br/>
+        <Link to='/sessionToken-login'>Session Token Login</Link><br/>
```

Comment:
nit: Mixing hypenation and camelCase?  ick

Reply:
this our test harness app.

Reply:
The URL already exists and is being used in the E2E tests. I just added a link on the page so we don't have to keep typing it in when we are testing manually.